src/main/scala/uk/gov/nationalarchives/dp/client/DataProcessor.scala

```diff
@@ -128,6 +128,18 @@ class DataProcessor[F[_]]()(implicit me: MonadError[F, Throwable]) {
       fromType[F](entityType, ref, title, description, deleted)
     }.sequence
 
+  def getIdentifiers(elem: Elem): F[Seq[IdentifierResponse]] = {
```

Comment:
This method is missing (`DataProcessorTest`) tests

Reply:
Knew I forgot something. I've added a couple in.